src/main/java/com/betternpchighlight/BetterNpcHighlightConfig.java

```diff
@@ -1613,10 +1613,11 @@ default String hideInstructions()
 		position = 13,
 		keyName = "tagStyleMode",
 		name = "Tag Style",
-		description = "Sets which highlight style list the NPC tagged is added too")
-	default tagStyleMode tagStyleMode()
+		description = "Sets which highlight style(s) the NPC tagged is added too"
+	)
+	default Set<tagStyleMode> tagStyleMode()
 	{
-		return tagStyleMode.TILE;
+		return Collections.emptySet();
```

Comment:
Trying to use `Collections.singleton` to instantiate a collection with 1 default tag style did not work. The result of the config call was still an empty set. I assume it still thinks there is an valid config available, even if nothing is selected. I'm not sure how to get around this yet, but I also havent looked into it.